index.bs

```diff
@@ -231,57 +231,57 @@ Given a task |task|, perform the following algorithm:
 
 2. Let |destinationRealms| be an empty set.
 
-3. Determine the set of JavaScript Realms to which reports will be delivered:
+3. Determine the set of <a href="https://tc39.github.io/ecma262/#sec-code-realms">JavaScript Realms</a> to which reports will be delivered:
 
-    For each environment settings object |settings| in task's script evaluation environment settings object set:
+    For each <a>environment settings object</a> |settings| in |task|'s [=task/script evaluation environment settings=] object set:
 
-    1. Let |topmostBC| be |settings|'s responsible browsing context's top-level browsing context.
-    2. Add |topmostBC|'s Window's relevant Realm to |destinationRealms|.
-    3. Let |descendantBCs| be |topmostBC|'s active document's list of descendant browsing contexts.
-    4. For each |descendantBC| in |descendantBCs|, add |descendantBC|'s Window's relevant Realm to |destinationRealms|.
+    1. Let |topmostBC| be |settings|'s <a>responsible browsing context</a>'s <a>top-level browsing context</a>.
+    2. Add |topmostBC|'s Window's <a>relevant Realm</a> to |destinationRealms|.
+    3. Let |descendantBCs| be |topmostBC|'s <a>active document</a>'s <a href="https://html.spec.whatwg.org/multipage/browsers.html#list-of-the-descendant-browsing-contexts">list of the descendant browsing contexts</a>.
```

Comment:
You shouldn't need a manual link here; `<a>` by itself should work. It doesn't because HTML isn't exporting this like it should. You can file a HTML bug, and/or use the link block.

Reply:
Done